@here/harp-mapview/lib/MapView.ts

```diff
@@ -3313,6 +3440,29 @@ export class MapView extends THREE.EventDispatcher {
                     return;
                 }
                 this.m_scene.add(light);
+                if ((light as any).isDirectionalLight) {
+                    const directionalLight = light as THREE.DirectionalLight;
+                    // This is needed so that the target is updated automatically, see:
+                    // https://threejs.org/docs/#api/en/lights/DirectionalLight.target
+                    this.m_scene.add(directionalLight.target);
+
+                    this.m_directionalLight = directionalLight;
+
+                    const enableDebugDirectionalLight = false;
+                    if (enableDebugDirectionalLight) {
```

Comment:
This code should be moved to the example, since everything we need is in the public `MapView` API